content/docs/user-guide/networking/multiplayer/split.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Client and Server Launchers with the Multiplayer Gem
+description: Build Open 3D Engine launchers with the Multiplayer Gem specifically targeting Clients, Servers or both.
+linkTitle: Separating Client and Server
+---
+
+The Multiplayer Gem supports builds that contain only client logic, only server logic, or both client and server logic. This allows users to create executables of smaller size by excluding unnecessary logic and dependencies. It also allows hiding potentially sensitive logic unique to one executable from the other. 
+
+* GameLauncher is a client only launcher
```

Comment:
Nit: feels like this list needs an introduction ie with this split we now three runtime executables: